logger/logger.go

```diff
@@ -87,13 +121,22 @@ func (l *Logger) Log(logType LogType, message string, info LogInfo) {
 	}
 }
 
+// Write implements the Write method of the io.Writer interface
+func (l *Logger) Write(p []byte) (n int, err error) {
```

Comment:
need this method to support https://github.com/teslamotors/fleet-telemetry/pull/157/files#diff-b7580450551eeabfcdda457f0467f7601dda587270a4ba1bef0d844f1da54786R23